### PR TITLE
Add git merging examples to the contribution guidelines

### PR DESCRIPTION
**Pull request by M4rtinK**
Add examples of how to properly merge branches to the contribution
guidelines:
- how to merge Fedora devel branch to master
- how to merge a GitHub pull request
- how to merge a topic branch manually

### CODE REVIEW

File: CONTRIBUTING [hunk 1]
```diff
@@ -99,3 +99,63 @@ Pull Request Review
 Please note that there is a minimum review period of 24 hours for any patch. The purpose of this rule is to ensure that all interested parties have an opportunity to review every patch. When posting a patch before or after a holiday break it is important to extend this period as appropriate.
 
 All subsequent changes made to patches must be force-pushed to the PR branch before merging it into the main branch.
+
+Examples
```
Comment (jkonecny12) [inline CONTRIBUTING]: Maybe to change ``Examples`` to ``Merging guidelines``?

The Contribution file is now so diverse that I don't know to which part are these examples belongs.

Comment (M4rtinK) [inline CONTRIBUTING]: We kinda have the merging guidelines stated above in the file already & these are really examples. So I guess it should be at least "Merging examples".

Comment (jkonecny12) [inline CONTRIBUTING]: "Merging examples" looks much better to me than "Examples".

**Comment by jkonecny12** [comment]
Looks good to me.

**Pull request merged**